docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
       - PHABRICATOR_URL=https://mozphab.dev.mozaws.net
       - PHABRICATOR_UNPRIVILEGED_API_KEY=api-123456789
       - TRANSPLANT_URL=https://stub.transplant.example.com
+      - DATABASE_URL=sqlite:////sqlite.db
```

Comment:
Not really an issue now, but, at some point we should start using postgres in the development environment since that what will be used in production.